e2e/e2eTestRunner.php

```diff
@@ -27,19 +35,19 @@
     exit($exitCode);
 }
 
+$symfonyStyleFactory = new SymfonyStyleFactory();
+$symfonyStyle =  $symfonyStyleFactory->create();
+
 $matchedExpectedOutput = false;
 $expectedOutput = trim(file_get_contents($expectedDiff));
 if ($output === $expectedOutput) {
-    $matchedExpectedOutput = true;
+    $symfonyStyle->success('End-to-end test successfully completed');
+    exit(Command::SUCCESS);
 }
 
-if ($matchedExpectedOutput === false) {
-    echo "\nEXPECTED:\n";
-    var_dump($expectedOutput);
-    echo "\nACTUAL:\n";
-    var_dump($output);
-    exit(1);
-}
+// print color diff, to make easy find the differences
+$consoleDiffer = new ConsoleDiffer(new Differ(), new ColorConsoleDiffFormatter());
+$diff = $consoleDiffer->diff($output, $expectedOutput);
+$symfonyStyle->writeln($diff);
```

Comment:
The former report only showed full texts and reported "some difference somewhere".

Now the diff only shows which exact line is different :+1:  e.g.

![image](https://user-images.githubusercontent.com/924196/148090760-194241b2-9004-4883-95c0-61b0ffe62dec.png)